pkg/queue/stats.go

```diff
@@ -41,36 +41,45 @@ const (
 )
 
 // NewStats instantiates a new instance of Stats.
-func NewStats(startedAt time.Time, reqCh chan ReqEvent, reportCh <-chan time.Time, report func(float64, float64, float64, float64)) {
+func NewStats(startedAt time.Time, reqCh chan ReqEvent, reportInterval time.Duration,
+	report func(float64, float64, float64, float64)) {
+	newStatsWithTicker(startedAt, reqCh, reportInterval, time.NewTicker(reportInterval).C, report)
```

Comment:
Do we need to stop `time.NewTicker(reportInterval).C` explicitly?

Reply:
Yes, I'll align with the activator code.

**Need** is a little harsh here though. The lifetime is bound to the queue-proxy anyway but being good citizen is probably not a bad idea.